+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Tue Aug  8 13:22:06 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL aide
+ pkgs=aide
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs aide --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpp08h5ln1
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpp08h5ln1
+ sed -i '$a/home DATAONLY' /etc/aide.conf
+ aide --init
Start timestamp: 2023-08-08 13:22:30 +0800 (AIDE 0.18.5)
AIDE successfully initialized database.
New AIDE database written to /var/lib/aide/aide.db.new.gz

Number of entries:	50746

---------------------------------------------------
The attributes of the (uncompressed) database(s):
---------------------------------------------------

/var/lib/aide/aide.db.new.gz
 MD5       : QWyAqNVOlF45cdAclUcmkA==
 SHA1      : JkJCj7QSQjD6XZrfuVMGxULiuKU=
 SHA256    : mW9vSpJiBIOXZuwQBSuCd+Sa/gHjm1O/
             JWPT09Umm38=
 SHA512    : KgTlCXgBEnYeZKLEnmD24KqyEkJ/FZbx
             PMQmIC4b8anwgy3bqVMmNbL6pEWnmf7d
             BbgILCMItNN/cx36BFCIBg==
 RMD160    : EUSeFHvPCzObvOVLKv+JBPVp/4c=
 TIGER     : Gnx9WQ4+XHPEzXgFsWt0l23g2XySv0NI
 CRC32     : A6Fzig==
 WHIRLPOOL : 88HSDRghrEo59Ig87A27UeOYtej5NjyC
             WirbbcXo8sV84cTh2T0PtKflETizgYU1
             KJuwQhz+8sizv4UDfOICog==
 GOST      : 6FYhAhw6U73W7vIw0sDbzAfw7i4oQZIX
             KUyTfM4vMWM=
 STRIBOG256: D6hbsZKKjtskZtd4//UInMCTCS2lsGiY
             bUd80mYrmVg=
 STRIBOG512: vY4NZtiEgQylEGwBrSxhUyt0deE9PD4b
             ZGhfeP5UZVuo0MfD7XAjqdojB/bqKpjW
             w6SoqXeBZJ0S2d7crt8Ymg==
 SM3       : nuRYa8g8nCcdUkLCpmvpjSZnZtgdCM/s
             Jekv441KYbE=


End timestamp: 2023-08-08 13:28:16 +0800 (run time: 5m 46s)
+ mv /var/lib/aide/aide.db.new.gz /var/lib/aide/aide.db.gz
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Tue Aug  8 13:28:16 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Aug  8 13:28:17 2023 - INFO  - Start to run test.
+ sed -i '$a/errorconf' /etc/aide.conf
+ CHECK_RESULT 0 0 0 'Modify aide configuration items: failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Modify aide configuration items: failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ aide --config-check
+ grep -i Error
  ERROR: /etc/aide.conf:177: syntax error, unexpected new line, expecting group (line: '/errorconf')
+ CHECK_RESULT 0 0 0 'Check aide configuration contain error: failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check aide configuration contain error: failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ sed -i '$d' /etc/aide.conf
+ CHECK_RESULT 0 0 0 'Delete aide configuration items: failed!'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Delete aide configuration items: failed!'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ aide --config-check
+ grep -i Error
+ CHECK_RESULT 1 1 0 'Check aide configuration not contain error: failed!'
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log='Check aide configuration not contain error: failed!'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Aug  8 13:28:18 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Tue Aug  8 13:28:18 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Tue Aug  8 13:28:19 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpp08h5ln1 ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpp08h5ln1
+ '[' 0 -ne 0 ']'
+ rm -rf /var/lib/aide
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Tue Aug  8 13:28:27 2023 - INFO  - End to restore the test environment.
